src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationSetting.cs

```diff
@@ -25,8 +34,8 @@ public ConfigurationSetting(string key, string value, string label = null)
         }
 
         /// <summary>
-        /// The primary identifier of a key-value.
-        /// The key is used in unison with the label to uniquely identify a key-value.
+        /// The primary identifier of a configuration setting.
```

Comment:
I've seen "Gets or sets " prefix being used for property documentation. We need to decide if we want to use it too.

Reply:
One vote against from me.   It's  the default template for properties and seems to get left a lot, but I question what value it provides.   Personally, I view the default case as being a read/write state and comment when that is not the case.